src/raven.js

```diff
@@ -341,10 +341,11 @@ Raven.prototype = {
             return;
         }
 
+        msg = msg + ''; // Make sure it's actually a string
         // Fire away!
         this._send(
             objectMerge({
-                message: msg + ''  // Make sure it's actually a string
+                message: msg,
```

Comment:
Mmm, in hindsight, I probably did this because in some version I did something else with `msg` so I wanted the conversion higher. I'll just restore this code.